skin-resque-frontend/src/App.js

```diff
@@ -3,7 +3,6 @@ import './App.scss';
 import Navbar from './components/Navbar/Navbar';
 import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
Module imports should occur before relative ones

Reply:
I fixed it